### DISCUSSION TITLE
DirectoryTree如何扩大点击范围

### DISCUSSION THREAD

**Discussion by fpsqdb** [ant-design/ant-design#50915] [Q&A]
https://ant.design/components/tree-cn#tree-demo-directory

类似vscode的目录树，点击整个item都可以选中或展开/收起，目前红框中点击不生效。
![image](https://github.com/user-attachments/assets/fe293ac3-0f63-4212-9d59-8f817032ae7b)

类似vscode的目录树，点击整个item都可以选中或展开/收起，目前红框中点击不生效。

**Comment by afc163** [comment]
这感觉是 bug。

**Reply by fpsqdb** [reply]
官网版本切换到3.x可以，4.x、5.x都不行
